### DISCUSSION TITLE
Option to show GPU, CPU temps only for monitoring temperatures when gaming or running full screen apps.

### DISCUSSION THREAD

**Discussion by STRunit** [seerge/g-helper#1448] [Ideas]
Is it possible to add an option to show the CPU & GPU temperatures only on display as an overlay or like floating widget. For example when we're gaming or running some full screen apps it would be nice to show the stats mentioned above on display corner. 

I use HWInfo with Rivaturner to monitor some stats for gaming but i usually only monitor the temperatures on them. My idea is use the HWInfo in the background without Rivaturner, but with the GHelper to see the temperatures only when gaming. And i can analyze the advanced system stats in HWinfo later when i am not gaming.

Anyway appreciate you a lot for making this awesome tool. Running smooth on my GX703HS.

PS: Excuse me if my English is bit rough.

**Comment by seerge** [comment]
@STRunit hello,  there is a lot of specialized apps for that, including ones bundled with Nvidia Experience or AMD Adrenaline drivers , i'm not a big fan of overloading apps with things things :) 

Rivatuner is also an option of course.

**Reply by STRunit** [reply]
Ok i got it, that's just a rough idea of mine. Been using Ghelper couple of weeks and seeing so much performance boosts compared to Armoury Crate lol. I'll contribute your project as i can when i get the chance. Huge thanks again ;)
